src/analysis/retail/paladin/holy/modules/checklist/Component.tsx

```diff
@@ -326,10 +331,14 @@ const HolyPaladinChecklist = ({ combatant, castEfficiency, thresholds }: Checkli
           </Trans>
         }
       >
-        <AbilityRequirement spell={SPELLS.DIVINE_STEED.id} />
-        <AbilityRequirement spell={SPELLS.DIVINE_PROTECTION.id} />
-        <AbilityRequirement spell={SPELLS.BLESSING_OF_SACRIFICE.id} />
-        <AbilityRequirement spell={SPELLS.LAY_ON_HANDS.id} />
+        <AbilityRequirement spell={TALENTS.DIVINE_STEED_TALENT.id} />
+        {combatant.hasTalent(TALENTS.DIVINE_STEED_TALENT.id)}
```

Comment:
This isn't really doing anything, you'd probably want this to be:
```suggestion
        {combatant.hasTalent(TALENTS.DIVINE_STEED_TALENT.id) && <AbilityRequirement spell={TALENTS.DIVINE_STEED_TALENT.id} />}
```

Reply:
Good suggestion, thank you.
Now included in PR